src/cljc/org/candelbio/multitool/core.cljc

```diff
@@ -471,6 +491,18 @@
     (doall thing)
     thing))
 
+(defn following-elt
+  [elt seq]
+  (cond (empty? seq) nil
+        (= (first seq) elt) (second seq)
+        :else (following-elt elt (rest seq))))
```

Comment:
No TCO in Clojure, might blow stack

Reply:
I rewrote some of those to use loop, but in general its not that much of a problem if you are working with reasonable sized seqs, if they are really so large that they will blow out the stack you probably don't want to be using naive fns like this, stack-safe or not,